sd-export/send-to-usb

```diff
@@ -0,0 +1,84 @@
+#! /usr/bin/python3
```

Comment:
Remove whitespace in shebang line. While it's technically permitted for whitespace to be present, it's inconsistent with the other scripts in the project:

```
$ grep -rIPoh '^#!\s?+.*$' | sort | uniq -c
     10 #!/bin/bash
     10 #!/bin/sh
      1 #!/usr/bin/env python3
      1 #! /usr/bin/python3
```

Recommend using `#!/usr/bin/env python3`, same as in the sd-proxy script. The whitespace in the shebang line also breaks autodetection of the file as a Python script, meaning the flake8 linting didn't catch it. :upside_down_face:  Expect a few linting errors once the shebang is updated.